src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String indentLine = "--------------------------------------------------";
+    private static Task[] tasks = new Task[100];
+    private static int numberOfTasks = 0;
+
+    public static void enterGreet() {
```

Comment:
Looks good! I like how the functions for the main code are split into different sub functions